dali/pipeline/pipeline.h

```diff
@@ -756,8 +771,18 @@ class DLL_PUBLIC Pipeline {
       return true;
     }
 
+    /**
+     * @brief Feeds the recently set inputs to the inputs that have `repeat_last` property
+     *
+     * @param owner       The pipeline
+     * @param fill_queue  If true, the inputs are fed `InputFeedCount(name)` times;
+     *                    otherwise it's fed once.
```

Comment:
Should it be plural?
```suggestion
     * @param fill_queue  If true, the inputs are fed `InputFeedCount(name)` times;
     *                    otherwise they're fed once.
```

Reply:
Yeah, I guess...